app/assets/stylesheets/bookings.scss

```diff
@@ -1,19 +1,79 @@
 @import "application";
 
-.thick-border {
-  border: 3px solid $my-gray;
-  padding-left: 1em;
-  padding-right: 1em;
-  padding-top: 1.5em;
-}
-
 .thin-wrap {
    border: 1px solid $my-gray;
    height: 6em;
 }
 
-.bold-text {
+.flight-details {
+  border-bottom: 1px solid $medium-gray;
+  color: $my-black;
+  font-family: $section-font;
+  font-size: 1.0em;
+  height: 3em;
+  padding: 1em;
+}
+
+.flight-details-header {
+  border-bottom: 1px solid $medium-gray;
+  font-family: $section-font;
+  font-weight: bolder;
+  text-align: center;
```

Comment:
Properties should be ordered border-bottom, font-family, font-weight, padding, text-align, text-transform